cmd/celestia-appd/cmd/root.go

```diff
@@ -108,7 +108,7 @@ func initAppConfig() (string, interface{}) {
 	// snapshots to nodes that state sync
 	srvCfg.StateSync.SnapshotInterval = 1500
 	srvCfg.StateSync.SnapshotKeepRecent = 2
-	srvCfg.MinGasPrices = fmt.Sprintf("0%s", app.BondDenom)
+	srvCfg.MinGasPrices = fmt.Sprintf("0.001%s", app.BondDenom)
```

Comment:
today I learned one can have fractions of 1 utia

Ref: 
- https://github.com/celestiaorg/celestia-app/blob/8b9c4c9d13fe0ccb6ea936cc26dee3f52b6f6129/testutil/network/network.go#L91
- https://github.com/celestiaorg/celestia-app/blob/8b9c4c9d13fe0ccb6ea936cc26dee3f52b6f6129/app/app.go#L95

Reply:
same!